custom_components/google_home/config_flow.py

```diff
@@ -34,11 +34,13 @@ class GoogleHomeFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
 
     def __init__(self) -> None:
         """Initialize."""
-        self._errors: Dict[str, str] = {}
+        self.errors: Dict[str, str] = {}
```

Comment:
Why renaming?

Reply:
I was thinking about unused variable convention but this is actually a private field. Python is a bit confusing, I'll revert this.